### DISCUSSION TITLE
Did the UI to delete a file change?

### DISCUSSION THREAD

**Discussion by d3netxer** [community/community#57328] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I seem to remember there was trash can icon to delete a file. After spending 10 minutes trying to find the internet for a solution. Now I need to click the button with the three dots, and even then the delete option in the submenu is hidden until I scroll down. This is unintuitive for me.

**Comment by greyesquaxar** [comment]
I had the same problem, thank you for telling me the solution, but you’re right, it’s not intuitive and that option should be less hidden

**Comment by mufthakherul** [comment]
maybe
